### ISSUE TITLE
[Feature Request] Setting to open comments in new tab

### ISSUE THREAD

**Issue by delendum** [rystaf/mlmym#40]
Mlmym is awesome - great work!

It would be nice if we could have a setting to open comments in a new tab by default when clicking on them:

https://i.imgur.com/d62lowV.png

Small quality of life stuff. :)

**Comment by Zamboniman** [comment]
Yup, I'd like this as well, as well as a dropdown list of subscribed communities. The things I got used to from RES.
